src/it/java/com/google/checkstyle/test/chapter4formatting/rule44columnlimit/ColumnLimit100Test.java

```diff
@@ -22,10 +22,8 @@
 import org.junit.jupiter.api.Test;
 
 import com.google.checkstyle.test.base.AbstractGoogleModuleTestSupport;
-import com.puppycrawl.tools.checkstyle.api.Configuration;
-import com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck;
 
-public class LineLengthTest extends AbstractGoogleModuleTestSupport {
+public class ColumnLimit100Test extends AbstractGoogleModuleTestSupport {
```

Comment:
Could you avoid using `100` in the naming? It the number is changed, we'll need to rename the file and update other places where it is used, no number - no further updates.

Reply:
As of current implementation of XdocPagesTest we require 100. But if @romani also agrees on your suggestion then I'll try some hack to avoid 100

Reply:
this name is dictated to chapter name, name is "matching" to name of chapter. 
If number change, it will be huge change for style guide.
I am ok to keep 100 in name of this test.